specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/examples/getDataContainer.json

```diff
@@ -0,0 +1,27 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspace123",
+    "api-version": "2020-09-01-preview",
+    "name": "datacontainer123"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/testrg123/providers/Microsoft.MachineLearningServices/workspaces/workspace123/datacontainers/datacontainer123",
+        "name": "datacontainer123",
+        "type": "Microsoft.MachineLearningServices/workspaces/datacontainers",
+        "properties": {},
```

Comment:
>"properties": {}, [](start = 8, length = 17)

Please provide example to understand how this will look like

Reply:
this is the example for getDataContainer, it returns a named container. It currently has very minimal meta-data at this time.